README.adoc

```diff
@@ -366,26 +365,26 @@ whenever a code change introduces a defect. JUnit and the JAX-RS Client API prov
 `src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceIT.java`
 ----
 
-The [hotspot=21]`@BeforeEach` and [hotspot=27]`@AfterEach` annotations indicate that this method runs either before or after the other test case.
+The [hotspot=Before]`@BeforeEach` and [hotspot=After]`@AfterEach` annotations indicate that this method runs either before or after the other test case.
 These methods are generally used to perform any setup and teardown tasks. In this case,
 the setup method creates a JAX-RS client, which makes HTTP requests to the `inventory` service. This
 client must also be registered with a JSON-P provider to process JSON resources.
 The teardown method simply destroys this client instance as well as the HTTP responses.
 
-The [hotspot=34-54]`testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
+The [hotspot=testFallbackForGet]`testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
 hostname before and after the `system` service becomes unavailable. Then, it asserts outputs from the two requests
 to ensure that they are different from each other.
 
-The [hotspot=33]`@Test` annotation indicates that the method automatically executes when your test class runs.
+The [hotspot=Test]`@Test` annotation indicates that the method automatically executes when your test class runs.
 
 In addition, a few endpoint tests have been included for you to test the basic functionality of the
 `inventory` and `system` services. If a test failure occurs, then you might have introduced a bug into
 the code.
 
 FaultToleranceIT.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright,javadoc tags=ft_testing']
```

Comment:
- don't think `tags=ft_testing` is needed here

Reply:
addressed.